calyx/src/passes/reset_insertion.rs

```diff
@@ -0,0 +1,53 @@
+use crate::ir::{
+    self,
+    traversal::{Named, Visitor},
+    LibrarySignatures,
+};
+use ir::traversal::{Action, VisResult};
+
+#[derive(Default)]
+/// Adds assignments from a components `clk` port to every
+/// component that contains an input `clk` port.
```

Comment:
Needs an update to say `reset` instead of `clk`? (Presumably this was copied from the clock insertion pass.)